README.md

```diff
@@ -6,46 +6,91 @@ schedgen
 
 This is a schedule image generator for streamer teams!
 
-How to use
-----------
+Software Requirements
+---------------------
+- [poetry](https://python-poetry.org): package manager (it should install summon as dev dependency)
+- Run `fc-match <FontName>` to get OS-hosted font file (change `<FontName>` for a font name of your choice)
 
-First of all, create a style file in TOML.
 
-Example:
+Usage
+-----
+
+```bash
+schedgen [OPTION]... WEEKDAY STREAMER...
+```
+
+Generates schedule announcement for streamers (check [example](./example.png)).
+
+Where OPTION may be one of following options:
+
+| OPTION        | Description                                             |
+|---------------|---------------------------------------------------------|
+| --background  | Background image relative path                          |
```

Comment:
Just "path", it could be relative or absolute, both are fine.

Reply:
ok